src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectScoresController.kt

```diff
@@ -21,68 +28,77 @@ import org.springframework.web.bind.annotation.RestController
+class ProjectScoresController(
+    private val projectScoreStore: ProjectScoreStore,
+) {
   @ApiResponse200
   @ApiResponse404
   @GetMapping
-  @Operation(summary = "Gets vote selections for a single project.")
+  @Operation(summary = "Gets score selections for a single project.")
   fun getProjectScores(
       @PathVariable("projectId") projectId: ProjectId,
   ): GetProjectScoresResponsePayload {
-    return GetProjectScoresResponsePayload(projectId, "testProject", emptyList())
+    val scoresByPhase = projectScoreStore.fetchScores(projectId)
+
+    val phaseScores =
+        scoresByPhase.map { (phase, scoreModels) ->
+          PhaseScores(
+              phase,
+              scoreModels.map { Score(it.category, it.modifiedTime, it.qualitative, it.score) },
+              ProjectScoreModel.totalScore(scoreModels))
+        }
+
+    return GetProjectScoresResponsePayload(phaseScores)
   }
 
   @ApiResponse200
   @ApiResponse404
   @PutMapping
   @Operation(
-      summary = "Upserts vote selections for a single project.",
+      summary = "Upserts score selections for a single project.",
       description =
           "Update the scores for the project phase. If the (project, phase, category) does not " +
               "exist, a new entry is created. Setting a `score` to `null` removes the score.")
   fun upsertProjectScores(
       @PathVariable("projectId") projectId: ProjectId,
-      @RequestBody payload: UpsertProjectScoresRequestPayload,
-  ): UpsertProjectScoresResponsePayload {
-    return UpsertProjectScoresResponsePayload(projectId, payload.phase, emptyList())
+      @RequestBody @Valid payload: UpsertProjectScoresRequestPayload,
+  ): SimpleSuccessResponsePayload {
+    projectScoreStore.updateScores(projectId, payload.phase, payload.scores.map { it.toModel() })
+
+    return SimpleSuccessResponsePayload()
   }
 }
 
 data class Score(
     val category: ScoreCategory,
-    @Schema(description = "Must be between -2 to 2. If `null`, a score has not been selected.")
-    val value: Int? = null,
+    val modifiedTime: Instant,
     val qualitative: String? = null,
-    val modifiedTime: Instant = Instant.now(),
+    @Schema(
+        description = "If `null`, a score has not been selected.", minimum = "-2", maximum = "2")
+    val value: Int? = null,
 )
 
 data class UpsertScore(
     val category: ScoreCategory,
-    @Schema(description = "Must be between -2 to 2. If set to `null`, remove the selected score.")
-    val value: Int? = null,
     val qualitative: String? = null,
-)
+    @field:Min(-2)
+    @field:Max(2)
+    @Schema(
+        description = "If set to `null`, remove the selected score.", minimum = "-2", maximum = "2")
+    val value: Int? = null,
+) {
+  fun toModel() = NewProjectScoreModel(category, null, qualitative, value)
+}
 
 data class PhaseScores(
     val phase: CohortPhase,
     val scores: List<Score>,
-    val systemScore: BigDecimal? = null,
+    val totalScore: BigDecimal? = null,
```

Comment:
Changed this name to reflect the fact that we're now calling this the "total score" per the requirements.